pkg/delete/delete.go

```diff
@@ -44,7 +48,7 @@ type configurationType = string
 type DeleteEntries = map[configurationType][]pointer.DeletePointer
 
 // Configs removes all given entriesToDelete from the Dynatrace environment the given client connects to
-func Configs(ctx context.Context, clients ClientSet, apis api.APIs, automationResources map[string]config.AutomationResource, entriesToDelete DeleteEntries) error {
+func Configs(ctx context.Context, clients ClientSet, _ api.APIs, automationResources map[string]config.AutomationResource, entriesToDelete DeleteEntries) error {
```

Comment:
Is it possible to remove `_ api.APIs` here?

Reply:
There are more than 30 usages of this function. I'll do it in a separate PR.